src/main/java/org/opensearch/jobscheduler/constant/CommonValue.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.jobscheduler.constant;
+
+public class CommonValue {
+
+    public static String EXTERNAL_ACTION_PREFIX = "cluster:admin/opendistro/js/";
```

Comment:
Why `opendistro`?

Reply:
I see, I am changing it to plugins. The reason to change as suggested by @saratvemulapalli is 

opendistro is part of ODFE: https://opendistro.github.io/for-elasticsearch/ which end of life.
AD security features were built in ODFE which is why it has legacy code
(https://amzn-aws.slack.com/archives/D03SBGET0BE/p1672437057031429)